tests/pr_test.go

```diff
@@ -170,8 +170,9 @@ func TestAgentsSolutionInSchematics(t *testing.T) {
 	} else {
 
 		options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
-			Testing: t,
-			Prefix:  "obs-agents",
+			TerraformVersion: "terraform_v1.9.0",
```

Comment:
I'm curious, was this needed? I started a thread in the #epx-project channel and they seemed to confirm that if it detected `>= 1.9.0` then schematics will create a workspace that meets that range if it supports it (meaning it should automatically create 1.9 workspace)

Reply:
ok I think its due to the way our test wrapper is creating the workspace. Terraform 1.6 is still the default, but if you did this outside of the test wrapper it should of automatically created a 1.9 workspace